src/features/broker-protection/comparisons/is-same-name.js

```diff
@@ -133,11 +143,58 @@ export function isSameName (fullNameExtracted, userFirstName, userMiddleName, us
     return false
 }
 
+/**
+ * Given the user's provided name, look for nicknames or full names and return a list
+ *
+ * @param {string | null} name
+ * @return {Set.<string>|Set<*>}
```

Comment:
IDEs tend to create overly wide types like this, but we should clean it up and just have `Set<string>`